uint16 volt
uint16 voltref
uint16 charge
uint16 discharge
uint32 timestamp